Sources/Core/Filter.swift

```diff
@@ -0,0 +1,52 @@
+//Idea for implementing the filters
+
+precedencegroup ForwardApplication {
+    associativity: left
+}
+//
+precedencegroup BackwardApplication {
+    associativity: right
+    higherThan: ForwardApplication
+}
+//
+precedencegroup ForwardComposition {
+  associativity: left
+  higherThan: ForwardApplication
+}
+
+precedencegroup BackwardComposition {
+  associativity: right
+  higherThan: BackwardApplication
+}
+
+infix operator |>: ForwardApplication
```

Comment:
The filter operators need some documentation and examples, as they are quite powerful tools.

Reply:
Do we maybe want a wrapper function that also does the explaining? like compose(this: A, into: B ) for >>> ?
And pipe(input: A, into: B) ?